docs/index.rst

```diff
@@ -3,22 +3,18 @@ AiiDA ABC tutorial
 
 .. _ABC_2021_Homepage:
 
-.. todo::
-
-   Update versions to ones from Quantum Mobile image
-
 +-----------------+-----------------------------------------------------------------------------------------------------------+
 | Related resources                                                                                                           |
 +=================+===========================================================================================================+
-| python packages | `aiida-core XXXX`_, `aiida-quantumespresso XXXX`_                                                         |
+| python packages | `aiida-core 1.3.0`_, `aiida-quantumespresso 3.0.0`_                                                       |
```

Comment:
The package versions probably won't correspond to these. I'm assuming they will still be the same as those from the BIGMAP tutorial.

```suggestion
| python packages | `aiida-core 1.4.3`_, `aiida-quantumespresso 3.2.1`_                                                       |
```

Reply:
Just checked the versions, and these are correct!